custom_components/bosch_alarm/translations/en.json

```diff
@@ -13,7 +14,16 @@
                 "data": {
                     "host": "Host",
                     "port": "Port",
-                    "password": "Automation Passcode \\ RSC+ Passcode"
+                    "password": "Automation Passcode",
+                    "installer_code": "Installer Code (Solution / AMAX only)"
```

Comment:
Is there a way we can add tooltips or some such to these to help users understand what code this referring to?

Reply:
![image](https://github.com/mag1024/bosch-alarm-homeassistant/assets/1862720/0196d85c-c0f1-43a8-8d24-d60cd6d151f2)

We can do this?